services/spd/trusty/trusty.c

```diff
@@ -385,11 +385,23 @@ static const spd_pm_ops_t trusty_pm = {
 	.svc_suspend_finish = trusty_cpu_suspend_finish_handler,
 };
 
+void plat_trusty_set_boot_args(aapcs64_params_t *args);
+
+#ifdef TSP_SEC_MEM_SIZE
+#pragma weak plat_trusty_set_boot_args
+void plat_trusty_set_boot_args(aapcs64_params_t *args)
+{
+	args->arg0 = TSP_SEC_MEM_SIZE;
```

Comment:
Why did you remove arg1 and arg2 from the equation?

Reply:
They are platform specific, I don't know where to get them from in generic code.

Reply:
In some implementations the previous level bootloader passes this information to the Trusted OS - memory aperture, uart port, etc. I think we should have that model and then sanity check the boot params in the SPD, instead of coming with another way of passing boot params.

Reply:
see comment below